packages/amazonq/src/lsp/chat/webviewProvider.ts

```diff
@@ -137,4 +142,11 @@ export class AmazonQChatViewProvider implements WebviewViewProvider {
         </body>
         </html>`
     }
+
+    async refreshWebview() {
```

Comment:
Is there a more general place we can put this? I imagine this is useful functionality for all webviews.